### PR TITLE
Make VTEC upgrade speed up borgs again.

### LINKED ISSUE #11138: VTEC's method of speed increase is outdated
The VTEC initially is supposed to set a cyborg's speed to a lower value.

For example, if a cyborg has speed = 0, then a vtec would set speed = -1.
The issue here is that most borgs have speed set equal to 0 and setting speed lower than 0 doesn't make the cyborg move faster. 

What DOES increase the cyborg's speed is "movement_speed_modifier". I've found that adding +0.5 to this value provides a solid speed increase to the cyborg. That being said, VTEC's method of increasing speed should adjust "movement_speed_modifier" and not "speed".

As a side note, resetting a borg should reset the "movement_speed_modifier" back to its original value.

### PR DESCRIPTION
**Pull request by Allspice**
Fixes #11138

### CODE REVIEW

File: code/game/objects/items/robot/robot_upgrades.dm [hunk 1]
```diff
@@ -3,6 +3,8 @@
 
 #define FAILED_TO_ADD 1
 
+/obj/item/borg/upgrade/var/vtec_bonus = 0.5
```
Comment (clusterfack) [inline code/game/objects/items/robot/robot_upgrades.dm]: Eh?

Comment (Allspice) [inline code/game/objects/items/robot/robot_upgrades.dm]: Not quite sure how to declare vars confined to a single dm honestly, and managing the bonus in both the reset and vtec module seemed bad.

Comment (JustSumBody) [reply code/game/objects/items/robot/robot_upgrades.dm]: Would probably want to do a define for that
`#define VTEC_BONUS 0.5` 
then `#undef VTEC_BONUS` at the bottom of the file

**Comment by jakmak** [comment]
How much does this speed up Borg's in comparison to normal borg speed?

**Comment by JustSumBody** [comment]
On that note, you can apply the vtec multiple times no?
Wouldn't this only remove the bonus once if you reset?

**Comment by Allspice** [comment]
PR removes multi VTEC

**Comment by clusterfack** [comment]
@jakmak Looks like it gives a 50% bonus to speed, or 33% reduction in time between steps. Pretty insanely high bonus.

**Comment by Allspice** [comment]
Pick a number, any number.

**Comment by clusterfack** [comment]
20%-25% should be noticeable but without making you go hyperspeed

**Comment by jakmak** [comment]
Eh for the insane material cost for a VTEC I feel this is balanced enough since upgrades are seldomly done for Borg's.

**Pull request merged**